docs/versioned_docs/version-v1.3.0/03-Guides/migration-guide-v1.0.md

```diff
@@ -0,0 +1,96 @@
+# Migration guide towards v1.0
+Starting from v1.0, there're some major breaking changes. To make it easier for you you migrate towards this new version, we have assembled an migration guide to help you in the process.
```

Comment:
```suggestion
Starting from v1.0, there are some major breaking changes. To make it easier for you to migrate towards this new version, we have assembled a migration guide to help you in the process.
```
This reads a bit weird.  I'd change it to not use a contraction.
See also: https://english.stackexchange.com/a/12869

Reply:
Yes, will update this.

Reply:
https://github.com/arcus-azure/arcus.messaging/pull/337/commits/3d5cd1df9e664f94bd6b9d1f7ceaa73b1180659e